phoenix-core/src/main/antlr3/PhoenixSQL.g

```diff
@@ -808,7 +808,14 @@ bind_expression  returns [BindParseNode ret]
     ;
     
 value_expression returns [ParseNode ret]
-    :   i=add_expression { $ret = i; }
+    :   i=pre_value_expression { $ret = i; }
```

Comment:
for example:
````
  json_column_a -> 'a'  ? json_column_b -> 'b' + 5
````
(skip type checking)
if we consider that first to get JSON value than plus 5 ,and than check existing JSON key/value(JSON array index), we should make that JSON value operation precedes JSON boolean operation and JSON value operation precedes arithmetic.